samples/drivers/auxdisplay/sample.yaml

```diff
@@ -7,5 +7,3 @@ tests:
     harness_config:
       fixture: fixture_auxdisplay
     platform_allow: nucleo_f746zg
-    integration_platforms:
```

Comment:
I beg to differ, I still have it hooked up, it requires the display be wired up as the overlay suggests but it is runnable. The point in this being an integration platform as far as I understand it is so CI builds is (i.e. during PRs) so it can be spotted if something has broken it. Correct me if I'm wrong @PerMac